src/modules/Lan.rb

```diff
@@ -703,11 +683,9 @@ def Import(settings)
       else
         NetworkService.use_wicked
       end
-      if Builtins.haskey(settings, "ipv6")
-        @ipv6 = Ops.get_boolean(settings, "ipv6", true)
-      end
 
-      LanItems.modified = true
+      @ipv6 = settings["ipv6"] || true
```

Comment:
`@ipv6 = settings["ipv6"] || true` NOOO!

That is the classical case where we must not use the `|| default` for supplying the default value. What if the user explicitly had `settings["ipv6"] = false`? That's why we must use `settings.fetch("ipv6", true)`.